civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -135,8 +135,9 @@ function _setup_modals() {
  * @param string $resources the name of the script file (extension included)
  * @param string $extension
  * @param string $page
+ * @param array $jsOptions
  */
-function _add_script_to_page($resources, $extension = NULL, $page = NULL) {
+function _add_script_to_page($resources, $extension = NULL, $page = NULL, $jsOptions = ['scope' => 'footer']) {
```

Comment:
does the additional param need to be an array at all? can't it be just `$scope = 'footer'`?

Reply:
yes! fixed it.